admin-console/src/main/webapp/src/pages/ApplicationPage.tsx

```diff
@@ -17,26 +17,32 @@ import { useParameterizedQuery } from 'react-fetching-library';
 import { useParams } from 'react-router';
 
 import { useCreateEventType } from '../services/CreateEventTypes';
+import { useDeleteEventType } from '../services/DeleteEventType';
 import { useApplicationTypes } from '../services/GetApplication';
 import { getBundleAction  } from '../services/GetBundleAction';
 import { useEventTypes } from '../services/GetEventTypes';
 import { EventType } from '../types/Notifications';
 
 type ApplicationPageParams = {
     applicationId: string;
+    eventTypeId: string;
```

Comment:
I didn't notice this before. I think this one is not used.
The `ApplicationPageParams` is used to get the types of the route params.

```suggestion
```